dockstore-common/src/main/java/io/dockstore/common/DescriptorLanguage.java

```diff
@@ -234,6 +234,21 @@ public FileTypeCategory getCategory() {
         }
     }
 
+    public static String getDefaultDescriptorPath(DescriptorLanguage descriptorLanguage) {
+        switch (descriptorLanguage) {
+        case CWL:
+            return "/Dockstore.cwl";
+        case WDL:
+            return "/Dockstore.wdl";
+        case NEXTFLOW:
+            return "/nextflow.config";
+        case GXFORMAT2:
+            return "/Dockstore.yml";
```

Comment:
This is a broader question, beyond the scope of this, but I think the default name should be something like "untitled.yml"... If I'm writing a real hosted workflow, which I've never done, I'd probably name it after its functionality, e.g. aligner.yml, aligner.wdl, etc. Calling it untitled makes it clearer I should change the name. This may not even be the place to solve it, assuming there's agreement, so ignore this, other than letting it percolate.

Reply:
Agreed, but more of a general usability issue I guess.